anomalib/deploy/optimize.py

```diff
@@ -65,14 +65,14 @@ def export_convert(
         input_names=["input"],
         output_names=["output"],
     )
+    export_path = os.path.join(str(export_path), export_mode)
```

Comment:
I would prefer if this line is above line 58 otherwise onnx model will live in the root directory (`export_path/model.onnx`) of exported models and metadata json will be under `export_path/onnx/meta_data.json`. But with this if the export_mode is openvino, we will get .bin and .onnx models under `openvino` folder. I am fine with this but I am not sure what you guys prefer.